spec/factories/git_repositories.rb

```diff
@@ -3,14 +3,15 @@
     initialize_with do
       repository_config = Hash.new
       repository_config['REPOSITORY_STORAGE'] =
-        "#{::Rails.root}/data/test/repos/test_repo"
+        "#{::Rails.root.to_s}/data/test/repos"
       repository_config['REPOSITORY_PERMISSION_FILE'] =
         REPOSITORY_STORAGE + '/conf'
       repository_config['IS_REPOSITORY_ADMIN'] = true
 
-      repo = Repository.get_class('git', repository_config)
-      repo.create(repository_config['REPOSITORY_STORAGE'])
-      repo.new(repository_config['REPOSITORY_STORAGE'])
+    repo = Repository.get_class('git', repository_config)
+
+    # Open the repo that was cloned from Gitolite in git_revision_spec.rb
+    repo.open("#{::Rails.root.to_s}/data/test/workdir")
```

Comment:
Redundant use of `Object#to_s` in interpolation.<br>Inconsistent indentation detected.